File: beginner/__2000__/2968/2968.php
<?php 

    list($voltas, $placas) = explode(" ", readline());

    $voltas = (int) $voltas;
    $placas = (int) $placas;

    $total_placas = $voltas * $placas;

    $results = [];

    for($i = 10; $i <= 90; $i += 10){
        $qnt_placas_por_porcentagem = ceil(($total_placas * $i) / 100);
        $results[] = $qnt_placas_por_porcentagem;
    }

    $output = implode(" ", $results);

    echo "$output\n";
?>

File: beginner/__2000__/2968/2968.cpp
#include <iostream>
#include <vector>
#include <cmath>
#include <string>

using namespace std;

int main(){

    int voltas, placas;
    cin >> voltas >> placas;

    double total_placas = stataic_cast<double>(voltas * placas);
    vector<int> results;

    for(int i = 10; i <= 90; i += 10){
        int qnt_placas_por_porcentagem = static_cast<int>(ceil((total_placas * i) / 100.0));
        results.push_back(qnt_placas_por_porcentagem);
    }

    string output;
    for(const int& result: results){
        output += to_string(result) + " ";
    }

    if(!output.empty()){
        output.pop_back();
    }

    cout << output << endl;

    return 0;
}